.codecov.yml

```diff
@@ -27,3 +27,4 @@ comment:
 
 ignore:
   - "*Errors.go"  # ignore error files
+  - "file.go"
```

Comment:
What's the reason for ignoring this file?

Reply:
I just did this because I don't have time to write json tests for every batch type, and I'm not sure how worth doing that is.

Reply:
yea, I thought we already had these JSON tests.

Reply:
Anyway, did a refactor so that this point is now moot